src/kvstore/kvstore_dist_server.h

```diff
@@ -32,6 +32,8 @@
 #include <functional>
 #include <future>
 #include <vector>
+#include "mxnet/c_api.h"
```

Comment:
i think stuff in include/mxnet/xxx would tend to be in <>

Reply:
Done